packs/equipment/cursebreak-bulwark.json

```diff
@@ -38,23 +38,24 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Cursebreaker Bulwark",
                 "predicate": [
-                    "raisedShield"
+                    "parent:equipped",
+                    "self:shield:raised",
+                    "item:type:spell"
                 ],
                 "selector": "saving-throw",
+                "slug": "cursebreak-bulwark-bonus",
                 "type": "circumstance",
                 "value": 1
             },
             {
-                "key": "FlatModifier",
-                "label": "Cursebreaker Bulwark (against curses)",
+                "key": "AdjustModifier",
+                "mode": "upgrade",
                 "predicate": [
-                    "raisedShield",
-                    "curse"
+                    "item:trait:curse"
                 ],
                 "selector": "saving-throw",
-                "type": "circumstance",
+                "slug": "cursebreak-bulwark-bonus",
```

Comment:
I made this an Adjust Modifier since the wording of the item says that the bonus increases against curses, but it's still a bonus against spells nevertheless. Let me know if you think it should stay as a separate Flat Modifier.